### ISSUE TITLE
Can't enable smart alarms on Mi Band 3

### ISSUE THREAD

**Issue by kisik21** [Freeyourgadget/Gadgetbridge#1384]
#### Before reporting a bug, please confirm the following:
- [x] I have read the [wiki](https://github.com/Freeyourgadget/Gadgetbridge/wiki), and I didn't find a solution to my problem / an answer to my question.
- [x] I have searched the [issues](https://github.com/Freeyourgadget/Gadgetbridge/issues), and I didn't find a solution to my problem / an answer to my question.
-  [x] _If you upload an image or other content, please make sure you have read and understood the [github policies and terms of services](https://help.github.com/articles/github-terms-of-service/#1-responsibility-for-user-generated-content)_ (not applicable)

#### Your issue is:
Cannot set smart alarms for mi band 3, the checkbox is absent. The default smart alarm becomes "dumb" when modified. Might be related to #864 and #1208 
#### Your wearable device is:
Xiaomi Mi Band 3
#### Your android version is:
8.0
#### Your Gadgetbridge version is:
0.31.3

**Comment by ashimokawa** [comment]
Does this work in mi fit?

**Comment by kisik21** [comment]
I don't have Mi Fit, should I install it? I heard it requires registration...

**Comment by vickyorlo** [comment]
I checked and Mi Fit indeed does not have smart alarms. However both Tools and Notify&Fitness seem to have the feature, and they both appear to not require the phone to be on for them to work. Maybe there's something to it.

**Comment by kisik21** [comment]
Closing issue as solved. Thanks @vickyorlo for testing this!

P. S. the absent alarm seems to be reserved for calendar events - I've just found this setting in Gadgetbridge.

**Comment by ashimokawa** [comment]
I tried to enable smart alarms multiple times but it had no effect back then (bip), will retry with mi band 3

**Comment by pinusc** [comment]
Hi, I know that as @kisik21 pointed out, this feature might not be a priority for GadgetBridge; what if I worked on it myself and opened a pull request? Would that be merged (provided of course the implementation is acceptable)?

Also I haven't quite been able to figure out why the support has been removed, and if it might be possible to enable it again or if a "smart alarm on-device emulation" (suggested in #864) would be needed.

I really hope we can get this to work - open source smart alarms are a dealbreaker for me!
